examples/examples-release-10/pom.xml

```diff
@@ -1,7 +1,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>io.kubernetes</groupId>
+    <parent>
+        <groupId>io.kubernetes</groupId>
+        <artifactId>client-java-examples-parent</artifactId>
+        <version>11.0.1-SNAPSHOT</version>
```

Comment:
I don't think we want this, do we? This makes these examples part of the overall release build (which definitely messed things up before.

Reply:
It doesn't get deployed because the parent has `<skip>true</true>` in the the deploy plugin. What else could go wrong?

Reply:
The maven release plugin looks for these strings and replaces them every time you run a release.

Reply:
The trick is to not make your modules break if the project version changes. I don't think this one will. It looks safe to me.